### PR TITLE
Or 1843 merge internal audit to main

### PR DESCRIPTION
**Pull request by nguyenzung**
For comparison, I will close. Please ignore

### CODE REVIEW

File: op-bindings/bindings/faultdisputegame.go [hunk 1]
```diff
@@ -28,6 +28,14 @@ var (
 	_ = event.NewSubscription
 )
 
+// TypesOutputRootProof is an auto generated low-level Go binding around an user-defined struct.
```
Comment (rlgns98kr) [inline op-bindings/bindings/faultdisputegame.go:31]: We have to delete it manually!

File: op-bindings/bindings/uniswapv3factory_more.go [hunk 2]
```diff
@@ -23,5 +23,5 @@ func init() {
 
 	layouts["UniswapV3Factory"] = UniswapV3FactoryStorageLayout
 	deployedBytecodes["UniswapV3Factory"] = UniswapV3FactoryDeployedBin
-	immutableReferences["UniswapV3Factory"] = false
+	immutableReferences["UniswapV3Factory"] = true
```
Comment (rlgns98kr) [inline op-bindings/bindings/uniswapv3factory_more.go:26]: We have to revert this manually!

**Comment by rlgns98kr** [comment]
rebased the main

**Review by theo-learner — APPROVED** [review]
Thank you so much. I have checked that we applied feedback from the internal audit, and all tests are working well.

**Review by rlgns98kr — APPROVED** [review]
Thank you very much! 🙇🏻‍♂️🙇🏻‍♂️

**Pull request merged**